frontend/src/pages/dashboard/Dashboard.tsx

```diff
@@ -1,9 +1,10 @@
+import firebase from 'firebase/app';
+import { Link, useNavigate } from 'react-router-dom';
 import './Dashboard.css';
```

Comment:
imports changed but rest of the code didnt?

Reply:
There might be some redundant moduls in previous import.